src/main/haskell/language-kore/test/parser/Data/Kore/ASTGen.hs

```diff
@@ -16,6 +16,13 @@ couple gen = do
         then return []
         else choose (0,3) >>= (`vectorOf` gen)
 
+couple1 :: Gen a -> Gen [a]
```

Comment:
Seems like too much overlapping with the above.  why not something like `pure (:) <*> gen <*> couple gen` ?